diff --git a/lib/library.cpp b/lib/library.cpp
index d48d59b..b98c25c 100644
--- a/lib/library.cpp
+++ b/lib/library.cpp
@@ -1299,7 +1299,7 @@ bool Library::isLeakIgnore(const std::string& functionName) const
     const  std::map<std::string, Function>::const_iterator it = functions.find(functionName);
     if (it != functions.cend())
         return it->second.leakignore;
-    return false;
+    return functionName != "*" ? isLeakIgnore("*") : false;
 }
 bool Library::isFunctionConst(const std::string& functionName, bool pure) const
 {
